File: lab9/khuynh9.java
//Kenny Huynh
//CSC-321
//Lab 9

import java.util.Scanner;

public class khuynh9
{
	public static void main(String[] args)
	{
		int randomnumber = 3;
		int userinput;

		System.out.print("Please guess a number between 1 - 20: ");
		Scanner input = new Scanner(System.in);

		userinput = input.nextInt();

		System.out.println("You have entered " + userinput);
		
		if(userinput == randomnumber)
		{
			int counter = 0;
			while (counter != randomnumber)
			{
				System.out.println("You won and guessed the number correctly!");
				counter++;
			}
		}
		else if (userinput > randomnumber)
		{
			System.out.println("You lost and your guess is higher than my number: " + randomnumber);
		}
		else if (userinput < randomnumber)
		{
			System.out.println("You lost and your guess is lower than my number: " + randomnumber);
		}
		else
		{
			System.out.println("THE IF STATEMENT FAILED");
		}
	}
}

File: lab9/khuynh9.c
//Kenny Huynh
//CSC-321
//Lab 9

#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

int main(void)
{
	//double input = 0;
	int input = 0;
	int rnumber = 3;
	bool valid = 0;

	printf("Please guess a number between 1 - 20: ");

	//scanf("%lf",&input);

	scanf("%d",&input);

	//printf("\nYou have entered: %lf",input);
	
	while (input > 20 || input < 1)
	{
		printf("You have entered an invalid number. \nPlease enter a valid number: ");
		scanf("%d",&input);
	}
	
	printf("You have entered: %d\n",input);

	if(input == rnumber)
	{
		int counter = 0;
		while (counter < rnumber)
		{	
			printf("You guessed correctly!\n");
			counter++;
		}
	}

}



